src/random.cpp

```diff
@@ -9,8 +9,16 @@
 #include <support/cleanse.h>
 #ifdef WIN32
 #include <compat.h> // for Windows API
-#include <wincrypt.h>
+#if defined(_MSC_VER) && defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0601
```

Comment:
checking `_WIN32_WINNT` is not necessary. The min version required to run Bitcoin Core is Vista which is the same as the required min version of `BCryptGenRandom`